decidim-initiatives/app/services/decidim/initiatives/pdf_signature_example.rb

```diff
@@ -1,8 +1,7 @@
 # frozen_string_literal: false
 
```

Comment:
```suggestion

# The hexapdf dependency needs to be added to the application bundle
# It can be done by running this command:
# bundle add hexapdf --version "~> 0.32.0"
```